weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -406,6 +427,9 @@ function buildCallsTableColumns(
     // Should probably have a custom filter here.
     filterable: false,
     sortable: false,
+    valueGetter: cellParams => {
```

Comment:
For a case like this, should the valueGetter have the same logic re: UNSET as renderCell, and then I think maybe we don't need renderCell?

Reply:
Yeah probably, although there are cases where the UI must be different because of rendering. Ill fix this one @jamie-rasmussen .